### ISSUE TITLE
Client fails to load profile after install

### ISSUE THREAD

**Issue by Diftraku** [keybase/client#6443]
When first installing Keybase from RPM and running `run_keybase` in a terminal, everything seems to work nicely, the squirrel comes up on the terminal and the GUI opens. Everything looks to be okay but the profile page comes up empty:
![Keybase GUI](https://i.imgur.com/I2m5LT1.png)

When navigating to devices tab, a prompt appears saying "Can't load devices". Other tabs seem to work just fine by the looks of it.

Quitting the client and then starting it again from the app drawer, the GUI loads up with the profile page and now I am presented with a PIN-prompt for the keybase passphrase. Entering the passphrase shows the policy change dialogue and confirming it leaves me with the empty profile page once more.

Devices tab lists all my devices and going back to the profile page finally loads the content as expected.

OS: Fedora 25 (64bit)
Keybase: version 1.0.20-20170329160043+6d18a6a
SELinux: Enforcing
Log ID after first start: a831fe31167bb188172db51c
Log ID after restart of the client: cdd49b7a83af6661f2d6081c

**Comment by cjb** [comment]
Hi, we had a server outage, that was probably your problem.  Everything should be back up now, please could you retry?

**Comment by Diftraku** [comment]
Yup, that seemed to be it. Doing a "clean" install immediately loaded the profile and PIN-promt.
